src/the_way/mod.rs

```diff
@@ -86,6 +86,13 @@ impl TheWay {
             }
             TheWayCLI::View { index } => self.view(*index),
             TheWayCLI::List { filters } => self.list(filters),
+            TheWayCLI::ImportGist { gist_url } => {
+                let gist_url = gist_url.clone();
+                let snippets = self.import_gist(&gist_url)?;
+                println!("Imported {} snippets from {}", snippets.len(), gist_url);
+                self.show_snippets(&snippets)?;
```

Comment:
This is nice for debugging but is probably not that user-friendly, especially if it's a gist with many files as then the terminal would suddenly be filled with text. I still like the separation of the `show_snippets` function, it'll be useful later for sure.